src/components/PageLink/PageLink.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  people: Person[],
+  person: Person,
+  isSelected: (person: Person) => boolean,
+};
+
+export const PageLink: React.FC<Props> = ({ people, person, isSelected }) => {
+  const mother = people.find((user) => user.name === person.motherName);
+  const father = people.find((user) => user.name === person.fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames(
+        { 'has-background-warning': isSelected(person) },
+      )}
+    >
+      <td>
+        <Link
+          to={`../${person.slug}`}
+          className={classNames(
+            { 'has-text-danger': person.sex === 'f' },
+          )}
+        >
+          {person.name}
+        </Link>
```

Comment:
Instruction: create the `<PersonLink person={person} />` and use it for all existing people;

also for mother and father obj